charts/tidb-operator/templates/admission/jobPatch/post-del-secret-job.yaml

```diff
@@ -0,0 +1,34 @@
+{{- if and (eq .Values.admissionWebhook.enabled true) (eq .Values.admissionWebhook.certProvider "autoGenerator") }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ template "chart.name" . }}-admission-delete
+  labels:
+    app.kubernetes.io/name: {{ template "chart.name" . }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/component:  admission-cert
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+"  "_" }}
+  annotations:
+    # This is what defines this resource as a hook. Without this line, the
+    # job is considered part of the release.
+    "helm.sh/hook": post-delete
+    "helm.sh/hook-delete-policy": hook-succeeded,hook-failed
+spec:
+  template:
+    metadata:
+      name: {{ template "chart.name" . }}-admission-delete
+    spec:
+      restartPolicy: Never
+      serviceAccountName: tidb-admission-webhook-sa
+      containers:
+        - name: "{{.Release.Name}}-job-del"
+          image: {{ .Values.admissionWebhook.autoGenerator.kubectl.image.repository }}:{{ .Values.admissionWebhook.autoGenerator.kubectl.image.tag }}
+          imagePullPolicy: {{ .Values.imagePullPolicy | default "IfNotPresent" }}
+          command:
+            - "sh"
+            - "-c"
+            - |
+              set -e
+              kubectl delete secrets -n {{ .Release.Namespace }} tidb-admission-webhook-certs || true
```

Comment:
could we retry?

Reply:
> could we retry?

I think this depends on is there any necessary to ensure the secret would be deleted after deleting the chart.

Reply:
Can we just add this in the document instead of doing it automatically?

Reply:
> Can we just add this in the document instead of doing it automatically? It's only a one time and deadly simple command.

sure